text/0000-dyn-error-generic-member-access.md

```diff
@@ -354,7 +354,7 @@ impl<'a> Request<'a> {
     }
 
     /// Get the `TypeId` of the requested type.
-    pub fn type_id(&self) -> TypeId {
+    fn type_id(&self) -> TypeId {
```

Comment:
Might be worth removing the method completely and accessing `self.type_id` at any call-sites.